fe/src/main/java/org/apache/doris/cluster/Cluster.java

```diff
@@ -128,6 +129,7 @@ public void addDb(String name, long id) {
         try {
             dbNames.add(name);
             dbIds.add(id);
+            dbNameToIDs.put(name, id);
```

Comment:
Do you need to rebuild the dbNameToIDs in readFields() function when replaying meta log?

Reply:
I think we needn't.

After Cluster readFields,  we will construct a new InfoSchemaDb and add it to Cluster immediately.

```
    public long loadCluster(DataInputStream dis, long checksum) throws IOException, DdlException {
        if (Catalog.getCurrentCatalogJournalVersion() >= FeMetaVersion.VERSION_30) {
            int clusterCount = dis.readInt();
            checksum ^= clusterCount;
            for (long i = 0; i < clusterCount; ++i) {
                final Cluster cluster = Cluster.read(dis);
                ...
                final InfoSchemaDb db = new InfoSchemaDb(cluster.getName());
                ...
                cluster.addDb(db.getFullName(), db.getId());
```